### ISSUE TITLE
CPU Memory consumption issue

### ISSUE THREAD

**Issue by Gaumera** [ywatanabe1989/custom_losses_pytorch#1]
While training model and weights of losses simultaneously is there any possibility to create a memory issue?  I have got runtime error like 'DataLoader worker is killed by a signal'. 
Can you explain how to merge this class with a model class to train the weights of the CNN model?
Thanks in advance.

**Comment by ywatanabe1989** [comment]
@Gaumera
Thank you for noticing the problem.

Although the name "multitasklloss_instance" includes "loss", multitaskloss_instance is just the same as other Neural Networks. This is because multitaskloss has learnable parameters.
The instance has to be put on either on CPU memory or GPU memory.

Do you use CPU for training both your CNN model and multitaskloss_instance?
If that is the case, shrinking the mini-batchsize might fix the problem.
